templates/web/prod/actions/Tools/videoEditor.html.twig

```diff
@@ -56,12 +59,30 @@
                                     {% set ratio = '' %}
                                 {% endif %}
 
-                                <video id="thumb_video" controls="" preload="auto" data-ratio="{{ ratio }}">
+                                {#Set the document Ratio #}
+                                {% for subdef in previewHtml5 %}
+
+                                    {% set width = subdef.get_width()  %}
+                                    {% set height = subdef.get_height() %}
+
+                                    {% if width and height %}
+                                        {% set rawRatio =  (width / height)|number_format(3, '.') %}
+                                        {% set rawRatioLength =  rawRatio|length %}
+                                        {% if subdef.get_name()=='preview' %}
+                                            {% set prevRatio = rawRatio|slice(0,rawRatioLength-1) %}
```

Comment:
2 decimals will always be enough, no need for 3 then slice to 2...
btw : removing last digit is wrong since 1.248 should become 1.25